cmd/api/src/api/middleware/middleware.go

```diff
@@ -149,13 +149,16 @@ func ContextMiddleware(next http.Handler) http.Handler {
 }
 
 func parseUserIP(r *http.Request) string {
-	if result := r.Header.Get("X-Forwarded-For"); result == "" {
-		log.Warnf("No data found in X-Forwarded-For header")
-		return r.RemoteAddr
+	res := ""
+	if ipAddress := r.Header.Get("X-Forwarded-For"); ipAddress != "" {
+		res += "X-Forwarded-For: " + ipAddress + "; "
 	} else {
-		result += "," + r.RemoteAddr
-		return result
+		log.Warnf("No data found in X-Forwarded-For, possible upstream misconfig?")
 	}
+
+	res += "Remote Address: " + r.RemoteAddr
+
+	return res
 }
```

Comment:
I think this is a result of the parent branch getting merged before you could rebase off of the changes made. Needs to be reverted

Reply:
I updated the functionality here to strip the port from `r.RemoteAddr`. It's just a randomly selected port that the client used to connect with and doesn't really add anything useful. I figured stripping it would avoid possible confusion.

Reply:
Good thinking on removing the port, I agree that it'll take away useless information that just adds confusion